server/models/Profile.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
```

Comment:
This code is repeated from the profile model branch. Let's base this branch. Read below 👍🏻 

[How to change the base branch of a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

Reply:
I changed the base to profile model branch and resolved the changes, is this correct?